Automatic Validation ended with:
> 2024-01-10T05:28:36.3932368Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [12.3, 12.3]]

(Automated response - build 543.)

--- a/manifests/n/Nvidia/CUDA/12.3.2/Nvidia.CUDA.installer.yaml
+++ b/manifests/n/Nvidia/CUDA/12.3.2/Nvidia.CUDA.installer.yaml
@@ -22,7 +22,5 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://developer.download.nvidia.com/compute/cuda/12.3.2/local_installers/cuda_12.3.2_546.12_windows.exe
   InstallerSha256: 3A4B476E1CD70EAABE42881B480F48557481D1EDE0AB5F33B2AF11B8903425B8
-AppsAndFeaturesEntries:
-  - DisplayVersion: "12.3"
 ManifestType: installer
 ManifestVersion: 1.5.0

